src/lib/bootloader/grub2base.rb

```diff
@@ -28,6 +28,7 @@
 
 module Bootloader
   # Common base for GRUB2 specialized classes
+  # rubocop:disable Metrics/ClassLength
```

Comment:
I'd prefer adding it to `.rubocop.yml` instead of polluting the code...

Reply:
it is already in master, so I just use it.